### ISSUE TITLE
Release Twisted web 0.6.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11220]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#1736|
|Type|task|
|Created|2006-05-22 00:05:25Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1736 1736
type__task task
reporter__radix radix
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1148256325000000 1148256325000000
changetime__1148260615000000 1148260615000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set status to `closed`|
|-|-|

(In [16916]) Version and News changes from the Twisted Web 0.6.0 release.

Author: Release Automation, radix
Reviewers: radix, exarkun
Fixes [#11220](https://github.com/twisted/twisted/issues/11220)

This fixes a few bugs in the XHTML documentation that were causing 
parse errors in Lore found during makeDocWeb.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
